src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public String goWork() {
+        return "Truck started its work!";
+    }

```

Comment:
The message returned by the `goWork` method should be more informative and indicate that it is specifically a `Truck` that is working. Consider including the type of machine in the message, for example: `Truck is starting its work!`.